lib/tetris.js

```diff
@@ -205,30 +240,43 @@ function runFnAtPiece (originalMatrix, piece, pos, rotation, runFunction) {
   }
   return matrix
 }
-
 function insertPiece (originalMatrix, piece, pos, rotation) {
   return changePiece(originalMatrix, piece, pos, pieceId, rotation)
 }
-
 function removePiece (originalMatrix, piece, pos, rotation) {
   return changePiece(originalMatrix, piece, pos, () => 0, rotation)
 }
-
-function copy (object) {
-  return JSON.parse(JSON.stringify(object))
-}
-
-function move (originalPlayer, newPos) {
-  let player = copy(originalPlayer)
-  player.piecePos = newPos
-  player.matrix = removePiece(player.matrix, player.piece, originalPlayer.piecePos, player.pieceRotation)
-  player.matrix = insertPiece(player.matrix, player.piece, player.piecePos, player.pieceRotation)
-  return player
-}
-
 function newPos (pos, direction) {
   return {x: pos.x + xDiff(direction), y: pos.y + yDiff(direction)}
 
   function xDiff (direction) { return direction === 'D' ? 1 : 0 }
   function yDiff (direction) { return (direction === 'R' ? 1 : 0) - (direction === 'L' ? 1 : 0) }
 }
+function randomPiece () {
+  return ['I', 'J', 'L', 'O', 'S', 'T', 'Z'][Math.floor(Math.random() * 7)]
```

Comment:
I researched a bit... I think the best way to go on this is use a random generator with seed, then we could pass it on initialState, it makes sense. The best I found was this two approaches: https://www.npmjs.com/package/pure-random https://jsperf.com/park-miller-vs-math-random
We can merge the code this way, and I make the changes after.